### PR TITLE
New tool: convertToLinearMesh

### PR DESCRIPTION
**Pull request by norihiro-w**
This PR adds the new tool `convertToLinearMesh` which will convert a nonlinear mesh to a linear mesh while keeping property values for base nodes.

### CODE REVIEW

File: MeshLib/MeshEditing/ConvertToLinearMesh.h [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#define MESHLIB_CONVERRTTOLINEARMESH_H_
+
+#include <memory>
+#include <string>
+
+namespace MeshLib
+{
+class Mesh;
+
+std::unique_ptr<MeshLib::Mesh> convertToLinearMesh(
```
Comment (endJunction) [inline MeshLib/MeshEditing/ConvertToLinearMesh.h]: Minor: A short comment about the details would be nice.

File: MeshLib/MeshEditing/ConvertToLinearMesh.h [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+#include <memory>
+#include <string>
+
+#include "MeshLib/Mesh.h"
+
+namespace MeshLib
+{
+
+/// Converts a non-linear mesh to a linear meash. All the mesh properties will
```
Comment (TomFischer) [inline MeshLib/MeshEditing/ConvertToLinearMesh.h]: Spelling: meash

File: MeshLib/MeshEditing/ConvertToLinearMesh.cpp [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    for (auto name : src_properties.getPropertyVectorNames())
+    {
+        auto const* src_prop = src_properties.getPropertyVector<double>(name);
+        if (!src_prop)
+            continue;
+        if (src_prop->getMeshItemType() != MeshLib::MeshItemType::Node)
+            continue;
+
+        auto const n_src_comp = src_prop->getNumberOfComponents();
+        // convert 2D vector to 3D. Otherwise Paraview Calculator filter does not recognize
```
Comment (TomFischer) [inline MeshLib/MeshEditing/ConvertToLinearMesh.cpp]: Maybe the converting will be valueable in other scenarios, too. Do you have plans to implement this as a separate tool?

Comment (norihiro-w) [inline MeshLib/MeshEditing/ConvertToLinearMesh.cpp]: sorry this conversion is already included in ProcessLib/LIE/Common/PostUtils.h, and shouldn't be here. I will remove it.

I don't have a plan yet. I can implement a CLI tool having this conversion. is it enough for you? I don't have a good idea about how to implement this to the process output.

Comment (TomFischer) [inline MeshLib/MeshEditing/ConvertToLinearMesh.cpp]: It is ok to remove it here.

**Pull request merged**